plugins/twimg.js

```diff
@@ -1,14 +1,20 @@
 ﻿var hoverZoomPlugins = hoverZoomPlugins || [];
 hoverZoomPlugins.push({
     name:'Twimg.com',
-    version:'0.1',
+    version:'0.2',
     prepareImgLinks:function (callback) {
         var res = [];
         hoverZoom.urlReplace(res,
             'img[src*="twimg.com"]',
             /_(normal|mini)/,
             ''
         );
+        hoverZoom.urlReplace(res,
+            'img[src*="twimg.com"]',
+            /_bigger/,
+            '_400x400'
+        );
+        $('div.css-1dbjc4n.r-1twgtwe.r-sdzlij.r-rs99b7.r-1p0dtai.r-1mi75qu.r-1d2f490.r-u8s1d.r-zchlnj.r-ipm5af').css('pointer-events', 'none');
```

Comment:
I'm not sure how stable this change would be. These class names look auto-generated during compile time so they might completely change every time Twitter updates its website. Is there any other way to find the element?

Reply:
Yeah, I thought so too, but rather I thought it changed on login (it doesn't).

But, you could be right, could change on recompile.

I did try to access it with the "nth-child" pseudo element, but apparently JQuery doesn't support that.

I'll look at it again to see if I can find a better way to access it.

Reply:
You don't have to find exact element with a single selector. Fetching multiple candidates and filtering through them using some custom javascript would be fine too.

Reply:
True.

Apparently I'm just crazy, I was wrong about JQuery not supporting the "nth-child" pseudo element. I may have been thinking about another project I'm working on that has an older version of JQuery.

Anyway, fixed the selector. Let me know what you think.

Reply:
Looks good to me, thank you for the contribution!

Reply:
No prob! I love this extension and I'm happy to help out!